language/stdlib/transaction_scripts/add_currency_to_account.move

```diff
@@ -5,10 +5,19 @@ use 0x1::Libra;
 use 0x1::Roles;
 use 0x1::Signer;
 
-/// Add a `Currency` balance to `account`, which will enable `account` to send and receive
-/// `Libra<Currency>`.
-/// Aborts with NOT_A_CURRENCY if `Currency` is not an accepted currency type in the Libra system
-/// Aborts with `LibraAccount::ADD_EXISTING_CURRENCY` if the account already holds a balance in
+/// # Sender Preconditions
```

Comment:
Nit: do we want to say "Signer Preconditions" to make this schema continue to work when we want to describe scripts with multiple signers?

Reply:
Signer preconditions I think makes sense, but  really like "script lifecycle" so will probably change to that to see if we like it if that sounds good